tests/bare/client.py

```diff
@@ -0,0 +1,28 @@
+import flwr as fl
+import numpy as np
+
+SUBSET_SIZE = 1000
+
+
+model_params = np.array([1])
+objective = 5
+
+# Define Flower client
+class DummyClient(fl.client.NumPyClient):
+    def get_parameters(self, config):
+        return model_params
+
+    def fit(self, parameters, config):
+        model_params = parameters
```

Comment:
This line is not necessary if you use `parameters` on the next line

Reply:
This in order to store the parameters in between rounds